src/sentry/runner/commands/run.py

```diff
@@ -534,74 +526,28 @@ def query_subscription_consumer(**options):
-def ingest_consumer(consumer_types, all_consumer_types, v2_consumer, **options):
+def ingest_consumer(consumer_type, **options):
     """
     Runs an "ingest consumer" task.
 
     The "ingest consumer" tasks read events from a kafka topic (coming from Relay) and schedules
     process event celery tasks for them
     """
-    if all_consumer_types:
-        if consumer_types:
-            raise click.ClickException(
-                "Cannot specify --all-consumer types and --consumer-type at the same time"
-            )
-        else:
-            consumer_types = set(ConsumerType.all())
-
-    if not all_consumer_types and not consumer_types:
-        raise click.ClickException("Need to specify --all-consumer-types or --consumer-type")
-
-    if v2_consumer:
-        from sentry.ingest.consumer_v2.factory import get_ingest_consumer
-
-        if len(consumer_types) != 1:
-            raise click.ClickException(
-                "Only a single `--consumer-type` allowed when using the `v2` ingest consumer"
-            )
-
-        # Our batcher expects the time in seconds
-        options["max_batch_time"] = int(options["max_batch_time"] / 1000)
-
-        type_ = consumer_types[0]
-        metrics_name = f"ingest_{type_}"
-        from arroyo import configure_metrics
+    from arroyo import configure_metrics
 
-        from sentry.utils import metrics
-        from sentry.utils.arroyo import MetricsWrapper
-
-        configure_metrics(MetricsWrapper(metrics.backend, name=metrics_name))
-
-        # Ignore these arguments which are only relevant for the old ingest consumer
-        options.pop("concurrency", None)
-
-        consumer = get_ingest_consumer(type_=type_, **options)
-        run_processor_with_signals(consumer)
-
-        return
-
-    # else:
-
-    from sentry.ingest.ingest_consumer import get_ingest_consumer
+    from sentry.ingest.consumer_v2.factory import get_ingest_consumer
     from sentry.utils import metrics
+    from sentry.utils.arroyo import MetricsWrapper
 
-    # Ignore these arguments which are only relevant for new ingest consumer
-    options.pop("strict_offset_reset", None)
-    options.pop("processes", None)
-    options.pop("input_block_size", None)
-    options.pop("output_block_size", None)
+    configure_metrics(MetricsWrapper(metrics.backend, name=f"ingest_{consumer_type}"))
 
-    concurrency = options.pop("concurrency", None)
-    if concurrency is not None:
-        executor = ThreadPoolExecutor(concurrency)
-    else:
-        executor = None
+    # TODO: Remove options from the old consumer
+    options.pop("concurrency", None)
+    options.pop("v2_consumer", None)
 
-    with metrics.global_tags(
-        ingest_consumer_types=",".join(sorted(consumer_types)), _all_threads=True
-    ):
-        consumer = get_ingest_consumer(consumer_types=consumer_types, executor=executor, **options)
-        run_processor_with_signals(consumer)
+    options["max_batch_time"] = int(options["max_batch_time"] / 1000)
```

Comment:
nit: If you like, you can keep the granularity and make it a float since that's what Arroyo takes for `max_batch_time`